django_app/settings.py

```diff
@@ -32,11 +32,11 @@
 
 SECRET_KEY = os.getenv("SECRET_KEY")
 
-CSRF_TRUSTED_ORIGINS = ['https://audio-discrimination-croudsource-dev']
+CSRF_TRUSTED_ORIGINS = ['https://audio-discrimination-croudsource-dev.fly.dev']
 
 DEBUG = os.getenv("DEBUG")
 
-ALLOWED_HOSTS = ['127.0.0.1','localhost','audio-discrimination-croudsource-dev'] # os.getenv("ALLOWED_HOSTS")
+ALLOWED_HOSTS = ['127.0.0.1','localhost','audio-discrimination-croudsource-dev','audio-discrimination-croudsource-dev.fly.dev'] # os.getenv("ALLOWED_HOSTS")
```

Comment:
Please take ALLOWED_HOSTS values from env file.

Reply:
@VivekWangoes thank you